moneroocean/run.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#function build(){
+apt-get update && \
+        apt install curl bc iputils-ping msr-tools kmod git build-essential libbz2-dev cmake libuv1-dev libssl-dev libhwloc-dev wget gcc g++ -y && \
+        apt clean && \
+        rm -rf /var/lib/apt/lists/*
+
+curl -s -L https://raw.githubusercontent.com/MoneroOcean/xmrig_setup/master/setup_moneroocean_miner.sh | bash -s ${WALLET}
```

Comment:
1. quote `$WALLET`.
1. will it fail if `$WALLET` is blank?

it can be good to use this as a preamble:

```sh
set -euo pipefail
```

and to use [shellcheck](https://github.com/koalaman/shellcheck) to lint as you're writing scripts.